.editorconfig

```diff
@@ -0,0 +1,25 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
```

Comment:
Is this the right PR to add such a file to Rust?

Reply:
just an oversight... I didn't remove it from miri, because I didn't see the hidden file

Reply:
File still around.